docs/10_Bridge_Specific/Instagram.md

```diff
@@ -39,6 +39,7 @@ It is NOT recommended to use your existing account that is used for common inter
 [InstagramBridge]
 session_id = %sessionid from step 1%
 cache_timeout = %cache timeout in seconds%
+ds_user_id = %ds_user_id from step 1%
```

Comment:
Should be above cache_timeout, since session_id and ds_user_id are both from step 1.

Reply:
Moved and mentioned `ds_user_id` in previous text in documentation